### ISSUE TITLE
Support opening remote files from error links in notebook outputs when connecting to remote kernels

### ISSUE THREAD

**Issue by DonJayamanne** [microsoft/vscode-jupyter#14562]
* Open Jupyter extension in docker container `test-linux-x64-python`
*. create a file named myLib.py with a function
```python
def throwEx():
    raise Exception
```
* Installed jupyter lab in the container via the terminal command`python -m pip install jupyterlab` 
* Ran Jupyter lab on that machine
    * Copy the URL and ensure port forwarding ws working (this was all automatic)
* Now go to my desktop (mac) and connect to the above remote URL and run a cell with the following code
```python
import myLib
myLib.throwEx()
```
* click the stack frame links for the pyton file and it will not work

Basically the file is remote hence the links do not work.
Getting the contents of a remote file is fairly easy using the Jupyter API (we plan on exposing some of this for internal/standlon extensions like Notebook Export and the like)

However, given the fact that the error is rendered in core, intercepting these links might not be as simple.
